htdocs/core/lib/functions.lib.php

```diff
@@ -4523,6 +4523,35 @@ function price2num($amount,$rounding='',$alreadysqlnb=0)
 	return $amount;
 }
 
+/**
+ *		Function to format a value into a defined format for French administration (no thousand separator & decimal separator force to ',' with two decimals)
+ *		Function used into accountancy FEC export
+ *
+ *		@param	float		$amount			Amount to format
+ *		@return	string						Chain with formatted upright
+ *
+ *		@see	price2num()					Revert function of price2fec
+ */
+function price2fec($amount)
```

Comment:
Can you move the method price2fec into functions2.lib.php ?
functions.lib.php is already very large and should contains only functions used everywhere in application.

Reply:
Done @eldy . Thanks for your report